<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WELCOME</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #424242;
background-color: rgb(0, 0, 0);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CreateAccount_Label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>981</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(253, 58, 115);
font: 75 60pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>CREATE ACCOUNT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_Button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>640</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(253, 58, 115);
	border-radius: 30px;
	font: 14pt &quot;Arial&quot;;
}
QPushButton:hover{
	background-color: rgb(253, 141, 174);
}
</string>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_LineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Email_LineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_LineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Age_LineEdit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>240</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Location_LineEdit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PhoneNo_lineEdit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>330</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Phone No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>570</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 44, 17);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Picture_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 2px solid rgb(253, 141, 174);
background: #424242;
font: 72pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Upload_Picture_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(253, 58, 115);
	border-radius: 20px;
	font: 11pt &quot;Arial&quot;;
}
QPushButton:hover{
	background-color: rgb(253, 141, 174);
}
</string>
    </property>
    <property name="text">
     <string>UPLOAD PICTURE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Gender_LineEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Gender</string>
    </property>
   </widget>
   <zorder>username_LineEdit</zorder>
   <zorder>CreateAccount_Label</zorder>
   <zorder>submit_Button</zorder>
   <zorder>Email_LineEdit</zorder>
   <zorder>Password_LineEdit</zorder>
   <zorder>Age_LineEdit</zorder>
   <zorder>Location_LineEdit</zorder>
   <zorder>PhoneNo_lineEdit</zorder>
   <zorder>error_label</zorder>
   <zorder>Picture_label</zorder>
   <zorder>Upload_Picture_button</zorder>
   <zorder>Gender_LineEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="myimqges.qrc"/>
 </resources>
 <connections/>
</ui>
